src/fastmcp/prompts/prompt_manager.py

```diff
@@ -78,7 +78,7 @@ async def _load_prompts(self, *, via_server: bool = False) -> dict[str, Prompt]:
             except Exception as e:
                 # Skip failed mounts silently, matches existing behavior
                 logger.warning(
-                    f"Failed to get prompts from mounted server '{mounted.prefix}': {e}"
+                    f"Failed to get prompts from {mounted.server.name} server mounted at '{mounted.prefix}': {e}"
```

Comment:
```suggestion
                    f"Failed to get prompts from server {mounted.server.name!r} mounted at {mounted.prefix!r}: {e}"
```

Reply:
Done. Added a bit more punctuation to make it very clear.